packages/react/src/components/Toast/index.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { FloatInAnimation, Position } from './FloatInAnimation';
+import { containerStyles, closeStyles, messageStyles } from './styles';
+import CloseIcon from './closeIcon';
+import { Portal } from '../Portal';
+
+type Type = 'success' | 'warning' | 'error' | 'default';
+type ToastOptions = { actions: any[] };
+
+interface ToastProps {
+  children: React.ReactNode;
+  variant?: Type;
+  position?: Position;
+  delay?: number;
+  options?: ToastOptions;
+}
+
+const DEFAULT_OPTIONS = { actions: [] };
+
+const Toast = ({
+  children,
+  variant = 'default',
+  position = 'bottom',
+  delay = 500,
+  options = DEFAULT_OPTIONS,
+}: ToastProps) => {
+  const [isVisible, setIsVisible] = React.useState<boolean>(true);
+
+  const close = () => setIsVisible(false);
+
+  const renderActions = () => null;
+
+  const getBodyContainerSx = () => {
+    switch (variant) {
+      case 'error':
+        return { background: 'red' };
+      case 'success':
+        return { background: 'green' };
+      case 'warning':
+        return { background: 'red' };
+      default:
+        return {};
+    }
+  };
+
+  if (!children) return null;
+
+  return (
+    <Portal>
```

Comment:
The `Popover` component would be a good reference for adding element to a dom. We should be able to use the `OverlayContainer`, `FocusScope`, and `Transition` components here for controlling mounting the Toast.